java-speech/google-cloud-speech/src/main/java/com/google/cloud/speech/v1/SpeechClient.java

```diff
@@ -447,6 +462,503 @@ public final UnaryCallable<RecognizeRequest, RecognizeResponse> recognizeCallabl
     return stub.streamingRecognizeCallable();
   }
 
+  // AUTO-GENERATED DOCUMENTATION AND METHOD.
+  /**
+   * Lists operations that match the specified filter in the request. If the server doesn't support
+   * this method, it returns `UNIMPLEMENTED`.
+   *
+   * <p>NOTE: the `name` binding allows API services to override the binding to use different
+   * resource name schemes, such as `users/&#42;/operations`. To override the binding, API services
+   * can add a binding such as `"/v1/{name=users/&#42;}/operations"` to their service configuration.
+   * For backwards compatibility, the default name includes the operations collection id, however
+   * overriding users must ensure the name binding is the parent resource, without the operations
+   * collection id.
+   *
+   * <p>Sample code:
+   *
+   * <pre>{@code
+   * // This snippet has been automatically generated and should be regarded as a code template only.
+   * // It will require modifications to work:
+   * // - It may require correct/in-range values for request initialization.
+   * // - It may require specifying regional endpoints when creating the service client as shown in
+   * // https://cloud.google.com/java/docs/setup#configure_endpoints_for_the_client_library
+   * try (SpeechClient speechClient = SpeechClient.create()) {
+   *   String name = "name3373707";
+   *   String filter = "filter-1274492040";
+   *   for (Operation element : speechClient.listOperations(name, filter).iterateAll()) {
+   *     // doThingsWith(element);
+   *   }
+   * }
+   * }</pre>
+   *
+   * @param name The name of the operation's parent resource.
+   * @param filter The standard list filter.
+   * @throws com.google.api.gax.rpc.ApiException if the remote call fails
+   */
+  public final ListOperationsPagedResponse listOperations(String name, String filter) {
```

Comment:
Do we need to expose these operations in the client explicitly?

Reply:
We expose IAMPolicy and Location RPCs in the client directly (https://github.com/googleapis/google-cloud-java/blob/d00a8493b7e18aa1792db8449f78d02258184668/java-functions/google-cloud-functions/src/main/java/com/google/cloud/functions/v1/CloudFunctionsServiceClient.java#L1216-L1218). Also given that we allows customers to access the OperationsClients (https://github.com/googleapis/google-cloud-java/blob/d00a8493b7e18aa1792db8449f78d02258184668/java-speech/google-cloud-speech/src/main/java/com/google/cloud/speech/v1/SpeechClient.java#L125-L126), I think it would make sense.